trunner/config.py

```diff
@@ -39,11 +39,15 @@ def resolve_phrtos_dir() -> Path:
 DEVICE_TARGETS = ['armv7m7-imxrt106x']
 
 # Port to communicate with hardware board
-DEVICE_SERIAL = "/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.4:1.1"
+DEVICE_SERIAL_PORT = "/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.4:1.1"
+DEVICE_SERIAL_BAUDRATE = 115200
 
 # DEVICE_SERIAL USB port address
 DEVICE_SERIAL_USB = "1-1.4"
 
+# Default psh prompt
+DEFAULT_PROMPT = '(psh)% '
+
```

Comment:
Do we need this var? When `prompt` keyword was implemented it was necessary but now is only used in the `handle` function.

Reply:
@jsarzy your remark was implemented